test/domain/model/contact_test.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:fluffychat/domain/model/contact/contact.dart';
+import 'package:fluffychat/domain/model/contact/hash_details_response.dart';
+
+void main() {
+  group('calLookupAddress', () {
+    group('calLookupAddress sha256', () {
+      const hashDetails = HashDetailsResponse(
+        algorithms: {'sha256'},
+        lookupPepper: 'matrixrocks',
+      );
+      test('calLookupAddress sha256 example 1', () {
```

Comment:
What is `example 1`, `example 2`, ...? Can you more detail on the case, on the input?

Reply:
done